packages/@lwc/wire-service/jest.config.js

```diff
@@ -10,4 +10,5 @@ module.exports = {
     ...BASE_CONFIG,
     displayName: 'lwc-wire-service',
     roots: ['<rootDir>/src'],
+    testEnvironment: 'jsdom',
```

Comment:
There is 1 spec in this package and it didn't seem to require any DOM functionality. Can this package still work with node as testEnvironment?
_I didn't try it locally. If you did and you noticed that it require jsdom, please ignore my comment_

Reply:
It is actually needed as wire service depends on `CustomEvent`.